e2e/app/page/cohort-search-page.ts

```diff
@@ -107,6 +107,7 @@ export default class CohortSearchPage extends AuthenticatedPage {
 
     const link = await this.waitForPhysicalMeasurementCriteriaLink(criteriaName);
     await link.click();
+    await waitWhileLoading(this.page);
```

Comment:
I expect this will address the flaky behavior in Cohort tests. Not removing 1 second sleep for now to see if flaky still happens.

Reply:
Just curious how adding this here would fix the flakiness? The error seems to happen several steps later in the test.

Reply:
Sounds good, thanks.